examples/hoomd-blue/umbrella_integration/integration.py

```diff
@@ -13,28 +13,30 @@
 from pysages.methods import UmbrellaIntegration
 
 
-param1 = {"A": 0.5, "w": 0.2, "p": 2}
+params = {"A": 0.5, "w": 0.2, "p": 2}
 
 
-def generate_context(**kwargs):
+def generate_context(A=0.5, w=0.2, p=2, **kwargs):
     hoomd.context.initialize("")
     context = hoomd.context.SimulationContext()
+
     with context:
         print(f"Operating replica {kwargs.get('replica_num')}")
-        system = hoomd.init.read_gsd("start.gsd")
+        hoomd.init.read_gsd("start.gsd")
 
-        hoomd.md.integrate.nve(group=hoomd.group.all())
-        hoomd.md.integrate.mode_standard(dt=0.01)
+        md.integrate.nve(group=hoomd.group.all())
```

Comment:
why can we omit the `hoomd.` here?
Did we change the imports?

Reply:
I changed the imports, but I can revert this change if you think it's clearer to leave the `hoomd.`

Reply:
It's OK. I was just a little confused since I missed the import change